src/pages/JobOwner/Home/Home.tsx

```diff
@@ -44,17 +41,21 @@ export default function ClientHome() {
             {t("Dashboard.buttons.seePosts")}
           </Button>
         </S.TitleContainer>
-        {mockPosts.map(({ createdAt, hired, messages, name, proposals }) => (
-          <div key={name} style={{ margin: "15px 0", width: "100%" }}>
-            <JobPost
-              createdAt={createdAt}
-              name={name}
-              hired={hired}
-              messages={messages}
-              proposals={proposals}
-            />
-          </div>
-        ))}
+        {posts ? (
+          posts.map(({ createdAt, title }) => (
+            <div key={title} style={{ margin: "15px 0", width: "100%" }}>
```

Comment:
Can the post id be the key? If we have posts with the same title React will complain